VEditorKit/Classes/VEditorNode.swift

```diff
@@ -235,6 +235,32 @@ open class VEditorNode: ASDisplayNode, ASTableDelegate, ASTableDataSource {
         }
     }
     
+    private func removeUnusedInternalASTableCellViewIfNeeds() {
+        // Bug Issue:  https://github.com/TextureGroup/Texture/issues/1407
+        let currentActiveTextASCellView = self.activeTextContainCellNode?.view.superview?.superview
+        
+        let invalidSubViews = self.tableNode.view.subviews.filter({ subView -> Bool in
+            
+            for visibleCell in self.tableNode.view.visibleCells {
+                if subView.frame == visibleCell.frame {
+                    return false
+                }
```

Comment:
I think if you use 'else', it will have little scopes.

Reply:
got it